library/aerospike

```diff
@@ -5,4 +5,5 @@
 3.14.1.8: git://github.com/aerospike/aerospike-server.docker@376c1940a6a30624305fc8e7e8acdefbb7867606
```

Comment:
Any particular reason to keep so many versions listed here?  Removing them here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Thanks for the info. I'll keep the 4.x only in that list on the next release.